### DISCUSSION TITLE
INF Waiting for login... - Tunnel not created

### DISCUSSION THREAD

**Discussion by fmika** [homeassistant-apps/app-cloudflared#250] [Q&A] [answered]
Can you help me?
I use my paid domain, I went through all necessary steps and on the cloudflare web I see my site with Active status.
But in the add-on log I see only these lines:

s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service init-banner: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
-----------------------------------------------------------
 Add-on: Cloudflared
 Use a Cloudflare Tunnel to remotely connect to Home Assistant without opening any ports
-----------------------------------------------------------
 Add-on version: 4.0.3
 You are running the latest version of this add-on.
 System: Home Assistant OS 9.3  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2022.11.2
 Home Assistant Supervisor: 2022.10.2
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub or forums.
-----------------------------------------------------------
s6-rc: info: service init-banner successfully started
s6-rc: info: service init-log-level: starting
s6-rc: info: service init-log-level successfully started
s6-rc: info: service init-cloudflared-config: starting
[17:07:34] INFO: Checking config for legacy options...
[17:07:35] INFO: Checking add-on config...
[17:07:36] INFO: Checking for existing certificate...
[17:07:36] NOTICE: No certificate found
[17:07:36] INFO: Creating new certificate...
[17:07:36] NOTICE: 
[17:07:36] NOTICE: Please follow the Cloudflare Auth-Steps:
[17:07:36] NOTICE: 
Please open the following URL and log in with your Cloudflare account:
https://dash.cloudflare.com/argotunnel?callback=https%3A%2F%2Flogin.cloudflareaccess.org%2F-fKxYASki0WlviLTpKaE4dtn35vcMj15rRH0AbEe6GU%3D
Leave cloudflared running to download the cert automatically.
2022-11-15T16:08:29Z INF Waiting for login...
2022-11-15T16:09:23Z INF Waiting for login...
2022-11-15T16:10:16Z INF Waiting for login...
2022-11-15T16:11:09Z INF Waiting for login...
2022-11-15T16:12:02Z INF Waiting for login...
2022-11-15T16:12:55Z INF Waiting for login...
2022-11-15T16:13:48Z INF Waiting for login...
2022-11-15T16:14:42Z INF Waiting for login...

**Comment by brenner-tobias** [comment, marked as answer]
Are you using the [local or remote tunnel](https://github.com/brenner-tobias/addon-cloudflared/blob/main/docs/tunnels.md) setup?
If you are using the remote managed tunnel, make sure to add the token from the Cloudflare Dashboard in the `tunnel_token` add-on option. 
If you are using the local tunnel, you have to open the link shown in the logs to authenticate with Cloudflare (see [here](https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#local-tunnel-add-on-setup-recommended)). One this is done, the tunnel will automatically connect.

**Comment by fmika** [comment]
Thanks for your fast reply.
It's my first Cloudflare Tunnel, so I am going to deepen my knowledge and to fix it. I will be bach to you soon.

________________________________
Od: Tobias Brenner ***@***.***>
Odesláno: středa 16. listopadu 2022 10:03
Komu: brenner-tobias/addon-cloudflared ***@***.***>
Kopie: František Mika ***@***.***>; Author ***@***.***>
Předmět: Re: [brenner-tobias/addon-cloudflared] INF Waiting for login... - Tunnel not created (Discussion #250)


Are you using the local or remote tunnel<https://github.com/brenner-tobias/addon-cloudflared/blob/main/docs/tunnels.md> setup?
If you are using the remote managed tunnel, make sure to add the token from the Cloudflare Dashboard in the tunnel_token add-on option.
If you are using the local tunnel, you have to open the link shown in the logs to authenticate with Cloudflare (see here<https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#local-tunnel-add-on-setup-recommended>). One this is done, the tunnel will automatically connect.

**Comment by fmika** [comment]
Success!
Now I know I was using local tunnel setup.
I decided to uninstall the add-in and to follow all steps from the beginning once again. All went smoothly.
I guess, the previous trouble was caused by that I did not wait with next steps until my domain change has been completed.
Thanks!
________________________________
Od: Tobias Brenner ***@***.***>
Odesláno: středa 16. listopadu 2022 10:03
Komu: brenner-tobias/addon-cloudflared ***@***.***>
Kopie: František Mika ***@***.***>; Author ***@***.***>
Předmět: Re: [brenner-tobias/addon-cloudflared] INF Waiting for login... - Tunnel not created (Discussion #250)


Are you using the local or remote tunnel<https://github.com/brenner-tobias/addon-cloudflared/blob/main/docs/tunnels.md> setup?
If you are using the remote managed tunnel, make sure to add the token from the Cloudflare Dashboard in the tunnel_token add-on option.
If you are using the local tunnel, you have to open the link shown in the logs to authenticate with Cloudflare (see here<https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#local-tunnel-add-on-setup-recommended>). One this is done, the tunnel will automatically connect.
